CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ ENHANCEMENTS:
 * Improved performance for large graphs when debug logs are not enabled. ([#1810](https://github.com/opentofu/opentofu/pull/1810))
 * Improved performance for large graphs with many submodules. ([#1809](https://github.com/opentofu/opentofu/pull/1809))
 * Added mutli-line support to the `tofu console` command. ([#1307](https://github.com/opentofu/opentofu/issues/1307))
+* Improve error handling for the Get command cover invalid args scenarios ([#1941](https://github.com/opentofu/opentofu/issues/1941))
```

Comment:
```suggestion
* Improved error handling for the Get command to cover invalid args scenarios ([#1941](https://github.com/opentofu/opentofu/issues/1941))
```

Reply:
Also, maybe one for @cam72cam I'm not sure if this is needed in the CHANGELOG or not. as it's just a test.

WDYT?

Reply:
I usually don't include this stuff in the changelog explicitly, but don't have a problem with entries like this.

Reply:
@cam72cam , @Yantrio thank you for your time and the review. Indeed it is a trivial change for a changelog entry. I have reverted the changelog entry and pushed the changes .  If you could please review it it would be great.